examples/src/StyleTester.js

```diff
@@ -1,15 +1,21 @@
 /* @flow */
-import React from 'react';
-import { StyleSheet, css } from '../../src/index.js';
+import * as React from 'react';
+import { StyleSheet, css } from '../../lib/index.js';
```

Comment:
The reason for importing `../../lib/index.js` instead of `../../src/index.js` is that `@babel/register` doesn't want to compile files from outside of its directory subtree.